keyboards/viendi8l/info.json

```diff
@@ -0,0 +1,97 @@
+{
+    "keyboard_name": "Viendi8L", 
+    "url": "https://viendi.co/", 
+    "maintainer": "gondolindrim", 
```

Comment:
```suggestion
    "keyboard_name": "8L", 
    "manufacturer": "Viendi",
    "url": "https://viendi.co/", 
    "maintainer": "gondolindrim", 
    "usb": {
        "vid": "0x8673",
        "pid": "0x0877",
        "device_version": "0.0.1"
    },
```

Reply:
Committed in [2d6b639](https://github.com/qmk/qmk_firmware/pull/17943/commits/2d6b6395a87f9692f71f5688cc3b282317139697)